cmd/mavenExecuteStaticCodeChecks.go

```diff
@@ -4,21 +4,44 @@ import (
 	"github.com/SAP/jenkins-library/pkg/command"
 	"github.com/SAP/jenkins-library/pkg/log"
 	"github.com/SAP/jenkins-library/pkg/maven"
+	"github.com/SAP/jenkins-library/pkg/piperutils"
 	"github.com/SAP/jenkins-library/pkg/telemetry"
+	"io"
 	"strconv"
 )
 
+type mavenStaticCodeChecksUtils interface {
+	Stdout(out io.Writer)
+	Stderr(err io.Writer)
+	RunExecutable(e string, p ...string) error
+
+	FileExists(filename string) (bool, error)
+}
+
+type mavenStaticCodeChecksUtilsBundle struct {
+	*command.Command
+	*piperutils.Files
+}
+
+func newStaticCodeChecksUtilsUtils() mavenExecuteIntegrationUtils {
+	utils := mavenExecuteIntegrationUtilsBundle{
```

Comment:
If you re-use mavenExecuteIntegrationUtilsBundle, why not also the interface?

Reply:
thx. copy paste issue